google/cloud/forseti/common/gcp_type/firewall_rule.py

```diff
@@ -861,7 +861,7 @@ def ips_in_list(ips, ips_list):
     for ip_addr in ips:
         if not ips_list:
             return False
-        if not any([ip_in_range(ip_addr, ips) for ips in ips_list]):
+        if not any([ip_in_range(ip_addr, addresses) for addresses in ips_list]):
```

Comment:
Also change `ips_list`, since the other `ips` are changed?

Reply:
We'd run into line length issues then. I improved the naming here.

FYI The reason I touched this is because the previously defined variable of `ips` was redefined in this list comprehension.